### ISSUE TITLE
Add download sizes similar to Android Studio Apk analyzer

### ISSUE THREAD

**Issue by kmadsen** [google/android-classyshark#189]
Is there a way to check the download size of native libraries (.so objects) for each build? Android Studio provides the download sizes for each of these, but we want to keep record of it internally. So we're looking to turn these values into JSON objects.

https://github.com/JetBrains/android/tree/master/apkanalyzer
![Screen Shot 2020-08-10 at 11 59 44 AM](https://user-images.githubusercontent.com/3021882/89819919-009d9900-db01-11ea-8bef-fdbcfaf41302.png)

**Comment by kmadsen** [comment]
I've dug into how the Android Studio plugin is working. I've made a jar similar to classy shark that uses their libraries that is working for our use case, but it would be good to use something with a broader audience. When investigating, here are the references identified.

- apkanalyzer documentation https://developer.android.com/studio/build/apk-analyzer
- android studio plugin for apkanalyzer https://github.com/JetBrains/android/tree/master/apkanalyzer
- android apkanalyzer source code https://android.googlesource.com/platform/tools/base/+/studio-master-dev/apkparser
- releases for apkanalyzer https://mvnrepository.com/artifact/com.android.tools.apkparser/apkanalyzer
- apk and app bundle size estimator  https://github.com/googlesamples/apk-patch-size-estimator
- app bundle documentation https://developer.android.com/guide/app-bundle

**Comment by borisf** [comment]
Always happy for pull requests :100:

**Comment by kmadsen** [comment]
Yeah can look into it soon. What are your thoughts on [kotlinx-cli](https://github.com/Kotlin/kotlinx-cli)?

Creating kotlin command line interfaces looks like it is becoming nice, and it's how I've set up our apkanalyzer

**Comment by borisf** [comment]
kotlinx-cli, don't have any opinions
